CMakeLists.txt

```diff
@@ -26,6 +26,35 @@ endif()
 
 file(GLOB _files *.fdm_material *.sig)
 
+# Determine version number
+find_package(Git)
+
+set(GIT_VERSION "Unknown")
+if(GIT_FOUND)
+    execute_process(
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        COMMAND ${GIT_EXECUTABLE} describe --tags --exact-match --dirty
+        OUTPUT_VARIABLE GIT_VERSION
+        ERROR_QUIET
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    if(GIT_VERSION)
+        set(CPACK_PACKAGE_VERSION "${GIT_VERSION}")
+    else()
+        execute_process(
+            WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+            COMMAND ${GIT_EXECUTABLE} describe --tags --dirty
+            OUTPUT_VARIABLE GIT_VERSION
```

Comment:
Same as above. Check `RESULTS_VARIABLE` before using the `OUTPUT_VARIABLE` to make it more robust.